server/src/main/java/io/deephaven/server/auth/CommunityAuthenticationModule.java

```diff
@@ -0,0 +1,124 @@
+import io.deephaven.proto.backplane.grpc.StorageServiceGrpc;
+import io.deephaven.proto.backplane.script.grpc.ConsoleServiceGrpc;
+import io.deephaven.server.session.TicketResolverBase;
+import io.grpc.health.v1.HealthGrpc;
+import org.apache.arrow.flight.impl.FlightServiceGrpc;
+
+import javax.inject.Singleton;
+
+@MapKey
+@interface ServiceGrpcKey {
+    String value();
+}
+
+
+@Module
+public class CommunityAuthenticationModule {
+    @Provides
+    @IntoMap
+    @ServiceGrpcKey(ApplicationServiceGrpc.SERVICE_NAME)
+    public ServiceAuthWiring bindApplicationServiceAuthWiring() {
+        return new ApplicationServiceAuthWiring.AllowAll();
+    }
+
+    @Provides
+    @IntoMap
+    @ServiceGrpcKey(ConfigServiceGrpc.SERVICE_NAME)
+    public ServiceAuthWiring bindConfigServiceAuthWiring() {
+        return new ConfigServiceAuthWiring.AllowAll();
+    }
+
+    @Provides
+    @IntoMap
+    @ServiceGrpcKey(ConsoleServiceGrpc.SERVICE_NAME)
+    public ServiceAuthWiring bindConsoleServiceAuthWiring() {
+        return new ConsoleServiceAuthWiring.AllowAll();
+    }
+
+    @Provides
+    @IntoMap
+    @ServiceGrpcKey(ObjectServiceGrpc.SERVICE_NAME)
+    public ServiceAuthWiring bindObjectServiceAuthWiring() {
+        return new ObjectServiceAuthWiring.AllowAll();
+    }
+
+    @Provides
+    @IntoMap
+    @ServiceGrpcKey(PartitionedTableServiceGrpc.SERVICE_NAME)
+    public ServiceAuthWiring bindPartitionedTableServiceAuthWiring() {
+        return new PartitionedTableServiceAuthWiring.AllowAll();
+    }
+
+    @Provides
+    @IntoMap
+    @ServiceGrpcKey(SessionServiceGrpc.SERVICE_NAME)
+    public ServiceAuthWiring bindSessionServiceAuthWiring() {
+        return new SessionServiceAuthWiring.AllowAll();
+    }
+
+    @Provides
+    @IntoMap
+    @ServiceGrpcKey(StorageServiceGrpc.SERVICE_NAME)
+    public ServiceAuthWiring bindStorageServiceAuthWiring() {
+        return new StorageServiceAuthWiring.AllowAll();
+    }
+
+    @Provides
+    @IntoMap
+    @ServiceGrpcKey(FlightServiceGrpc.SERVICE_NAME)
+    public ServiceAuthWiring bindFlightServiceAuthWiring() {
+        return new FlightServiceAuthWiring.AllowAll();
+    }
+
+    @Provides
+    @IntoMap
+    @ServiceGrpcKey(HealthGrpc.SERVICE_NAME)
+    public ServiceAuthWiring bindHealthAuthWiring() {
+        return new HealthAuthWiring.AllowAll();
+    }
+
+    @Provides
+    @Singleton
+    public TableServiceContextualAuthWiring bindTableServiceContextualAuthWiring() {
+        return new TableServiceContextualAuthWiring.AllowAll();
+    }
+
+    @Provides
+    @Singleton
+    public InputTableServiceContextualAuthWiring bindInputTableServiceContextualAuthWiring() {
+        return new InputTableServiceContextualAuthWiring.AllowAll();
+    }
+
+    @Provides
+    @Singleton
+    public TicketResolverBase.AuthTableTransformation bindAuthTableTransformation() {
+        // in community this is an identity transformation
+        return (table) -> table;
```

Comment:
I would create an Identity enum in AuthTableTransformation and return it here.

Reply:
or use Function.identity()?